bitnami/keycloak/values.yaml

```diff
@@ -788,6 +788,9 @@ serviceAccount:
   ## If not set and create is true, a name is generated using the fullname template
   ##
   name: ""
+  ## @param serviceAccount.automountServiceAccountToken Auto-mount the service account token in the pod
+  ##
+  automountServiceAccountToken: true
```

Comment:
Please set the default value to `false`.

As you mentioned in your description, it is safer to not automount the service account token, and we are trying to follow a 'secure by default' approach.

>By disabling the automount, potential attackers cannot access the Kubernetes API on behalf/through the pod.